core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1273,4 +1303,26 @@ public String getDefaultValue() {
     Object value = DEFAULT_VALUES.get(this);
     return value != null ? value.toString() : null;
   }
+
+  /**
+   * @param name the name of a property key
+   * @return if this property key is deprecated
+   */
+  public static boolean isDeprecated(String name) {
```

Comment:
If taking PropertyKey as parameter here, then how can I get the name of the PropertyKey, like just "CONF_DIR". The DEFAULT_KEYS_MAP can only offer the key of "alluxio.conf.dir", which does not I want.